torchreid/data/datasets/dataset.py

```diff
@@ -57,6 +57,7 @@ def __init__(
             self.combine_all()
 
         self._cut_train_ids(min_id_samples)
+        self.data_counts = self.get_data_counts(self.train)
```

Comment:
Please remove the duplicate line of code upper (line 54).

Reply:
Fixed